includes/database/class-edd-db-table-discountmeta.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Discount Meta Table: EDD_DB_Table_Discountmeta class
+ *
+ * @package Plugins/EDD/Database/Object/Meta
+ */
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+if ( class_exists( 'WP_DB_Table' ) ) :
+/**
+ * Setup the global "edd_discountmeta" database table
+ *
+ * @since 3.0.0
+ */
+final class EDD_DB_Table_Discountmeta extends WP_DB_Table {
```

Comment:
Aren't we namespacing all this new code?

Reply:
`WP_DB_Table` is a base class that won't need namespacing. We could if we really want to be overly cautious, but changing or aliasing it later is a non-breaking change also. 50/50.